docsrc/getting_started/jetpack.rst

```diff
@@ -86,8 +86,8 @@ Install pip and required python packages:
 
 .. code-block:: sh
 
-   # install pytorch from nvidia jetson distribution: https://developer.download.nvidia.com/compute/redist/jp/v61/pytorch
-   python -m pip install torch https://developer.download.nvidia.com/compute/redist/jp/v61/pytorch/torch-2.5.0a0+872d972e41.nv24.08.17622132-cp310-cp310-linux_aarch64.whl
+   # install pytorch from nvidia jetson distribution: https://pypi.jetson-ai-lab.dev/jp6/cu126/
+    pip3 install torch torchvision torchaudio --index-url https://pypi.jetson-ai-lab.dev/jp6/cu126/
```

Comment:
Looking at the index, there are jp62 builds for PyTorch 2.7.0 and CUDA 12.8? What are the rules behind these builds / Jetson compute stack if you know? My understanding was jp62 was CUDA 12.6 / TensorRT 10.3

Reply:
if you look inside index you can see pytorch 2.6.0 stack:
for 12.6 that is jetpack 6 is 12.6 jetpack 7 is coming with ubuntu 24.04 so this is why we move everything to ubuntu 24.04
https://pypi.jetson-ai-lab.dev/jp6/cu126